### PR TITLE
More deck api 128361161

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: source/includes/_endpoint-deck.md [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+```
+
+The following attributes are editable on this endpoint:
+
+ * name
+ * description
+ * is_public
+ 
+The other attributes are read-only and will 400 if the payload tries to change
+them.
```
Comment (nealrichardson) [inline source/includes/_endpoint-deck.md]: We say something different below, in the PATCH entity discussion. Which is correct?

File: source/includes/_endpoint-deck.md [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+    "template": "{\"query\": {\"measures\": \"Object with keyed measures functions\", \"dimensions\": \"Array of variable references\", \"weight\": \"Optional weight variable URL\"}, \"display_settings\": {}, \"query_environment\": {\"filter\": \"Array of filter urls\"}}"
+}
+
+```
+
+Each tuple on the slides catalog contains the following keys:
+
+Name | Type | Description
+---- | ---- | -----------
+analysis_url | url | Points to the analysis contained on this slide
```
Comment (nealrichardson) [inline source/includes/_endpoint-deck.md]: What if there's more than one analysis?

Comment (jjdelc) [inline source/includes/_endpoint-deck.md]: It is mentioned below that the first(oldest available) analysis is linked. Future analyses are available on the catalog but the original payload doesn't change.

File: source/includes/_endpoint-deck.md (hunk 1 above)
Comment (jjdelc) [inline source/includes/_endpoint-deck.md]: Both, the other paragraph adds the permission details for is_public

**Pull request merged**